install_octopus.sh

```diff
@@ -1,66 +1,100 @@
 #!/bin/bash
 # This script prepares the download of octopus in the right location given the version number, location to untar / clone and install prefix
 # example run:
-# $ ./install_octopus.sh 13.0 /opt/octopus /home/user/octopus-bin
-# $ ./install_octopus.sh develop /opt/octopus
-# Consider runing install_dependencies.sh first to install all the dependencies on a debian based system
+# $ ./install_octopus.sh --version 13.0 --download_dir /opt/octopus --install_dir /home/user/octopus-bin
+# $ ./install_octopus.sh --version develop --download_dir /opt/octopus
+# Consider running install_dependencies.sh first to install all the dependencies on a debian based system
 
-# exit on error and print each command
-set -xe
+
+
+# Function to display script usage
+usage() {
+  echo "Usage: $0 [--version <version_number>] [--download_dir <download_location>] [--install_dir <install_prefix>]"
+  echo "Options:"
+  echo "  --version <version_number>      Specify the version number of Octopus (e.g., 13.0, develop)"
+  echo "  --download_dir <download_location>   Specify the download location for Octopus source"
```

Comment:
I think it would be convenient and intuitive if the download directory would be the current directory by default. That's the normal behaviour when you run a `install` command of some type, and would reduce what you need to type in interactive use.

Reply:
updated to use `pwd` as the default download directory. The only mandatory entry now is the version